src/main/resources/init.sh

```diff
@@ -77,11 +77,11 @@ status)
                 exit 0
             else
                 printf "%s\n" "Warning, Pid $PID appears to not correspond to service $SERVICE"
-                exit 0
             fi
         fi
-        printf "%s\n" "Process dead but pidfile exists"
-        exit 1
+        printf "%s\n" "Process dead but pidfile exists. Cleaning up."
+        rm -f $PIDFILE
+        exit 3
```

Comment:
Mostly for consistency with the other 'service not running' branch below. Happy to change it back if the contract is meant to be `0 if running, non-0 otherwise`, though that leaves no way to message internal errors.

From a API perspective, it seems strange to expose whether the pid file existed or not; that's an implementation detail of the script, rather than anything actually useful to any caller.